### ISSUE TITLE
Windows Sym and Hardlink Fix, How do I suggest it?

### ISSUE THREAD

**Issue by CiniqueL** [beetbox/beets#3105]
### Problem
So, I'm not great with Github; I'm pretty sure I have to do a pull request and maybe other things for this, but I was so excited I thought I'd at least mention it and maybe get basic directions on how to suggest the code change:

I edited the __init__.py file and got symbolic and hardlinking to work on Windows.

How do I add/suggest the code to be added to the master branch?

**Comment by sampsyo** [comment]
Hi! Yes, please! A pull request is the right way to show us what you've done.

**Comment by CiniqueL** [comment]
Before I attempt to do a pull request I have a question:
If I deleted code and I do the pull request and you accept, will the code be deleted in the master branch?

I created a fork and I edited two of the files. The setup (to add a new module requirement for creating Windows links, pywin32), and I used it in the __init__ within the util folder.

But, I am not sure how to only use the functions I created if the system is Windows; so, I just exchanged the Unix `try` functions for the Windows ones. You could probably figure that part out faster than I.

**Comment by sampsyo** [comment]
> If I deleted code and I do the pull request and you accept, will the code be deleted in the master branch?

Not quite sure I understand—but once a PR is merged into this repository, it doesn't matter what you do with your copy. :)

**Comment by CiniqueL** [comment]
I think that answers my question...
Let me double-check. This is my first time contributing to a project; so, I'm not certain how it works.

I was asking if the changes I make will affect your main repository...?

In your `util/__init__` there are the functions for `link` and `hardlink` that use a `try` statement with `os.symlink` - which doesn't work for Windows.

I replaced those sections with code that works for Windows, but I don't want your master code to be overwritten.

**Comment by sampsyo** [comment]
When you open a pull request, that lets us see your code. We (the maintainers) can then decide whether we want to include it in master. If we do that, then it does change the main beets code. But we're not required to do so—hence the "request" part of the name.
